sonar-dotnet-shared-library/src/test/java/org/sonar/plugins/dotnet/tests/CoverageReportImportSensorTest.java

```diff
@@ -136,8 +136,6 @@ public void analyze() throws Exception {
     SensorContextTester context = computeCoverageMeasures(false);
     assertThat(context.lineHits("foo:Foo.cs", 2)).isEqualTo(1);
     assertThat(context.lineHits("foo:Foo.cs", 4)).isEqualTo(0);
-    assertThat(context.coveredConditions("foo:Foo.cs", 1)).isEqualTo(2);
```

Comment:
do you have to remove these lines, or can you just update them?

Reply:
I've added the lines back and updated the expected values to null